.coafile

```diff
@@ -1,7 +1,7 @@
 [default]
 bears = LineLengthBear, SpaceConsistencyBear
 files = **.js, **.css, **.html, **.yml, **.json
-ignore = **.min.js, **.min.css, tweetheatmap/public/lib/**, webtweets/js/bootstrap/**, webtweets/css/bootstrap.css, js/typeahead.bundle.js, webtweets/js/bootstrap.js
+ignore = **.min.js, **.min.css, tweetheatmap/public/lib/**, webtweets/js/bootstrap/**, webtweets/css/bootstrap.css, js/typeahead.bundle.js, webtweets/js/bootstrap.js, **/node_modules/**, **/index_bundle.js, WorldMoodTracker/index.html
```

Comment:
See [here](https://github.com/fossasia/apps.loklak.org/pull/189/files#diff-675bbc883c350abe040e9b7515e8a8b4R15) and [here](https://github.com/fossasia/apps.loklak.org/pull/189/files#diff-675bbc883c350abe040e9b7515e8a8b4R6).